### DISCUSSION TITLE
v5 to v6

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#632] [Q&A (Help)] [answered]
I have three v5 based projects with no customizations and I tried to convert one of them to v6-CoreUIv2 as advised by https://backpackforlaravel.com/docs/6.x/upgrade-guide but for now it failed because they disappeared all my menus :(
 
Has anyone already converted from v5 to v6?

Is there any guide to switch from v5+CoreUI to v6+Tabler ?

**Comment by pxpm** [comment, marked as answer]
Hey @blondie63 like we stated there in the upgrade guide, if you have A LOT of customization in core backpack files (like auth etc) changing to tabler may be a little bit difficult because of changes in the HTML structure. 

What we advise users to do, and how we have done in our projects, was to upgrade using coreuiv2, there are minimal changes you need to do, like Step 21 https://backpackforlaravel.com/docs/6.x/upgrade-guide#step-21 , maybe it's what's missing here ? 

We did indeed upgraded projects to v6 with tabler, for example our demo was upgraded to v6 with tabler. 

Let me know how it goes.

Cheers

**Reply by blondie63** [reply]
Do you remember this?

https://github.com/Laravel-Backpack/community-forum/discussions/616#discussioncomment-6691214

**Reply by pxpm** [reply]
Thanks, I aknowledge the problem now. 

I was wasn't able to reproduce it, because I always selected the theme like we say you should do in the upgrade guide. 

I opened an issue to discuss with @tabacitu how we should deal with this. 

I agree with you that allowing user to don't select a theme and then breaking is not the correct behavior we intended. 

https://github.com/Laravel-Backpack/CRUD/issues/5274

**Reply by blondie63** [reply]
@pxpm I did everything on a new branch so I have no problem going back!
But do you think the problem is related to the fact that I only installed the theme after ?

**Reply by blondie63** [reply]
Yes you are right, I skipped Step 21 :(
At this point i don't know if it's better to start upg from scratch or try to fix..
A question, during normal install v6, when i choose theme CoreUIv2, what happen on my folder resources/views/vendor/backpack/base/inc ? 
because here i've all my menu and some little customizations..
<img width="326" alt="BKP6-upg-step21" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/1a1ab5ff-7d2b-47bc-ae1b-5357fa9a2fc4">

**Reply by pxpm** [reply]
We don't delete any files or change any files. 
I don't think you need to start from scratch but maybe you will need to update some of them, for example the step 21 move what you have inside `sidebar` to `menu_items`. 

I think some of those files you shouldn't be overwriting, for example the `scripts.blade.php`.  If your intention was to add some custom javascript it's the best pratice to create your custom file, and include it in the scripts section of the config.

I think we also deleted the `full_screen_loader.blade.php` for example, but if you use it in your project, you can keep it.

You should probably check the files you have overwritten in your project vs the coreui-v2 files https://github.com/Laravel-Backpack/theme-coreuiv2/tree/main/resources/views . Do any adjustments needed and you should finish the update with success. 

Alternatively, since you mentioned tabler, this would be the same process to update to tabler, so if you want to go directly to tabler you need to update your overwritten files to match the tabler structure https://github.com/Laravel-Backpack/theme-tabler/tree/main/resources/views

Let me know how it goes.

Cheers

**Reply by blondie63** [reply]
ok thanks, i think to try update to tabler directly :)
